AudioKit/Operations/Signal Generators/Oscillators/AKFMOscillator.h

```diff
@@ -40,21 +40,39 @@ NS_ASSUME_NONNULL_BEGIN
 + (instancetype)presetDefaultOscillator;
 
 /// Instantiates the oscillator with 'stun-ray' type values
-+ (instancetype)presetStunRay;
++ (instancetype)initWithPresetStunRay;
```

Comment:
You probably shouldn't have a class method's name that starts with `init...`, that's pretty much against adopt conventions, and I would expect to create problems with Swift.